Sources/Engine/Graphics/Texture.cpp

```diff
@@ -1614,12 +1614,6 @@ void ProcessScript_t( const CTFileName &inFileName) // throw char *
   INDEX i=0;
   FOREACHINLIST( CFileNameNode, cfnn_Node, FrameNamesList, it1)
   {
-    if( i != 0) {   // we have to skip first picture since it has already been done
```

Comment:
Why did you remove this test?

Reply:
Because i is set to 0 on line 1614 and as a result it will always be 0 (never !=0).